nexus/tests/integration_tests/endpoints.rs

```diff
@@ -2535,16 +2535,21 @@ pub static VERIFY_ENDPOINTS: LazyLock<Vec<VerifyEndpoint>> = LazyLock::new(
                 ],
             },
             VerifyEndpoint {
-                url: "/v1/system/update/repository?file_name=demo-repo.zip",
+                url: "/v1/system/update/repositories?file_name=demo-repo.zip",
                 visibility: Visibility::Public,
                 unprivileged_access: UnprivilegedAccess::None,
-                allowed_methods: vec![AllowedMethod::Put(
-                    // In reality this is the contents of a zip file.
-                    serde_json::Value::Null,
-                )],
+                allowed_methods: vec![
+                    // the query param is only relevant to the put
+                    AllowedMethod::Put(
+                        // In reality this is the contents of a zip file.
+                        serde_json::Value::Null,
+                    ),
+                    // get doesn't use the query param but it doesn't break if it's there
+                    AllowedMethod::Get
+                ],
```

Comment:
I tried to put these in two separate entries so I could leave the query param off the get URL, but it doesn't work because it wants all calls to a give path in one entry -- it expected GET to 405 when it was testing the PUT by itself.